pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -31,6 +32,7 @@ buildPythonPackage rec {
 
   outputs = [ "out" "dev" ];
 
+  nativeBuildInputs = stdenv.lib.optional (!isPyPy) python.pythonForBuild.pkgs.cffi;
```

Comment:
Splicing doesn't seem to work here. Anyone know why?

Reply:
https://github.com/NixOS/nixpkgs/issues/49526

Reply:
If this is always the case we are going to have a bit of a problem, because all the `nativeBuildInputs` in case of Python packages are then incorrect.